src/content/docs/building-with-kessel/how-to/report-resources.mdx

```diff
@@ -140,3 +140,166 @@ sent, it will miss the notification. Listening should be one of the first things
 because LISTEN/NOTIFY is a long-lived operation that can block other operations on the same connection. You should also 
 aim to minimize connections in a way that each new LISTEN does not produce a new connection, but rather reuses an 
 existing one.
+
+## Kafka Consumer Strategies
+As a service provider, you need a reliable way to replicate data changes from your systems to the Fabric. 
+A Kafka consumer offers a robust and scalable pattern to solve this problem. A consumer reads messages from Kafka topics by subscribing to topic partitions. 
+The consuming application then processes the messages to apply those changes.
+
+This guide outlines the key challenges your consumer should address and provides
+some high-level strategies for building a robust solution.
+
+We built our consumer in Go using the [confluent-kafka-go](https://github.com/confluentinc/confluent-kafka-go) client library.
+
+### Why Use a Kafka Consumer for Replication?
+When replicating data, a Kafka consumer solves the core problem of reliably processing asynchronous messages.
+It decouples your primary service from the replication process, meaning an issue in the replication pipeline won't 
```

Comment:
Somewhat a nitpick but our consumer isn't decoupled from our application and could absolutely cause performance issues, so it depends on how a consumer is built. I see you call this out at the bottom so maybe we just need some reorganization or rephrasing here.

Reply:
o yup, absolutely yeah it *can* decouple but depends on how its built and deployed. I was thinking of the perspective of a standalone consumer pod here but I'll be more general. Will followup 👍